code/modules/research/techweb/all_nodes.dm

```diff
@@ -140,7 +140,7 @@
 	prereq_ids = list("practical_bluespace", "high_efficiency")
 	design_ids = list("bluespace_matter_bin", "femto_mani", "triphasic_scanning", "tele_station", "tele_hub", "quantumpad", "launchpad", "launchpad_console",
 	"teleconsole", "bag_holding", "bluespace_crystal", "wormholeprojector", "bluespace_pod")
-	research_cost = 2500
+	research_cost = 15000

```

Comment:
x6 increase? Not sure if this is a meme PR or not.

Reply:
> boh > t4 parts> wormhole gun

> x6 of medical record computers

yeah I'd say that's fair.